discovery_engine/lib/src/domain/models/active_search.dart

```diff
@@ -26,7 +26,7 @@ part 'active_search.g.dart';
 class ActiveSearch with _$ActiveSearch {
   @HiveType(typeId: searchTypeId)
   const factory ActiveSearch({
-    @HiveField(0) required String queryTerm,
+    @HiveField(0) required String searchTerm,
```

Comment:
❔I know too little about the hive db but this change doesn't require any mirgration, does it?

Reply:
good question. according to the hive docs, renaming (while keeping the `HiveField` number the same) should be fine.

Reply:
yes, it should be fine, field name is not important as long as it uses the same `HiveField` identifier.